tools/dockerfiles/Dockerfile.1.10.canary

```diff
@@ -0,0 +1,27 @@
+FROM golang:1.10-alpine
+
+LABEL a01.product="azuresdkforgo"
+LABEL a01.index.schema="v2"
+LABEL a01.env.AZ_SUBSCRIPTION_ID="secret:sp.go.subscriptionid"
+LABEL a01.env.AZ_TENANT_ID="secret:sp.tenant"
+LABEL a01.env.AZ_CLIENT_ID="secret:sp.go.clientid"
+LABEL a01.env.AZ_CLIENT_SECRET="secret:sp.go.clientsecret"
+LABEL a01.env.AZ_LOCATION="centraluseuap"
+
+# install additional stuff alpine does not have
+RUN apk add --no-cache git bash
+
+# get everything needed to build and run tests
+COPY . /go/src/github.com/Azure-Samples/azure-sdk-for-go-samples
+RUN go get github.com/golang/dep/cmd/dep
+WORKDIR /go/src/github.com/Azure-Samples/azure-sdk-for-go-samples
+RUN dep ensure
+
+# add get_index and droid
+WORKDIR /
+RUN mkdir /app
+RUN mv go/src/github.com/Azure-Samples/azure-sdk-for-go-samples/tools/get_index /app/get_index
+RUN mv go/src/github.com/Azure-Samples/azure-sdk-for-go-samples/tools/a01droid /app/a01droid
+RUN go run /go/src/github.com/Azure-Samples/azure-sdk-for-go-samples/tools/list/list.go > /app/test_index
```

Comment:
Channel all the commands in the following format to reduce the number of the layers in the image so as to reduce the image size.
```
RUN command1 && \
    command2 && \
    ...
    last_command
```

Reply:
Fixed this with the labels too :)